### PR TITLE
Rename work item

### PR DESCRIPTION
**Pull request by OR13**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jwt/pull/125.html" title="Last updated on Jul 12, 2023, 5:49 PM UTC (c6b17ee)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jwt/125/227499a...c6b17ee.html" title="Last updated on Jul 12, 2023, 5:49 PM UTC (c6b17ee)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -829,6 +831,70 @@ <h2><code>application/vc+ld+sd-jwt</code></h2>
...
   </section>
+  
+
+  <section id="json-web-token-claim-registrations">
+    <h3>JSON Web Token Claims</h3>
+    <p>
+      This section retained deprecated registered claim names, 
+      that have been submitted to the Internet Engineering Steering Group
+      (IESG) for review, approval, and registration with IANA in the
+      "JSON Web Token Claims Registry".
```
Comment (TallTed) [inline index.html:842]: As nearby, it seems there should be a link (direct or reference) to the "JSON Web Token Claims Registry"
```suggestion
      This section retains deprecated registered claim names, 
      that were previously submitted to the Internet Engineering Steering
      Group (IESG) for review, approval, and registration with IANA in the
      "JSON Web Token Claims Registry".
```

**Review by TallTed — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (OR13) [reply index.html]: I probably should not have merged this, given this PR is attempting to implement a WG resolution... its possible I will need to manually reimplement this change depending on what happens next.

Comment (iherman) [reply index.html]: Indeed. As i said in my comment: as soon as we start down the road of the change, everything else should remain frozen...

**Comment by iherman** [comment]
@OR13, I indeed did not give all the details in #115 because I thought it was obvious that the full respec structure of the spec must be changed. My bad. I will create a separate PR with the way I believe the respec must be. I will also cc the webmaster to that PR to be on the safe side, and I actually will have a question to him there.

To avoid any mess up, I would propose:

- We close this PR without a merge
- The yml change for the actions that you had in this PR should be made separately and asap; better do it as a separate step to avoid any action happening in subsequent PR-s. (It is actually unnecessary, imho, to go through a PR for that one, just do the change directly on the repo.)
- I realized that we must change the repo _before_ the publishing. The fact is that respec refers and, more importantly, pulls in issues based on the github reference, and that must be the final one. So just ping me when the repo change can be done.
- I will set the publication date to July 20; we should be able to handle all the steps by then.

**Comment by OR13** [comment]
https://github.com/w3c/vc-jwt/commit/ba80df799deaa64b03a392cb1d7b5ae6f2f4cce8 turned off auto publish.

**Comment by OR13** [comment]
Closing as directed in https://github.com/w3c/vc-jwt/pull/125#issuecomment-1633943671

**Pull request closed without merging**